samples/png.cc

```diff
@@ -21,7 +21,7 @@
 
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wweak-vtables"
-#include "third_party/lodepng/lodepng.h"
+#include "lodepng.h"
```

Comment:
This should be lodepng/lodepng.h. I'm pretty sure the linter is going to give an error with this being a directory-less include path.

Reply:
done